### PR TITLE
Back 800 update configuration

### PR DESCRIPTION
**Pull request by RomanWlm**
Refresh configuration of WD
- Default Vault Explorer V3 
- update_wallet_config is not an option anymore (reload at startup)
- disable sync token by default

### CODE REVIEW

**Review by mbasire-ledger — CHANGES_REQUESTED** [review]

File: src/it/resources/application.conf [hunk 1]
```diff
@@ -134,9 +134,9 @@ explorer = {
       }
       {
         currency = ethereum_classic
-        host = "https://explorers.api.live.ledger.com"
+        host = "https//etc.explorers.vault.ovh.ppr.ldg-tech.com"
```
Comment (mbasire-ledger) [inline src/it/resources/application.conf]: `host = "https://etc.explorers.vault.ovh.ppr.ldg-tech.com"`

**Review by mbasire-ledger — APPROVED** [review]

File: src/it/resources/application.conf [hunk 2]
```diff
@@ -139,7 +139,7 @@ explorer = {
         currency = ethereum_classic
         host = "https://explorers.api.vault.ledger.com"
         host = ${?WALLET_ETC_EXPLORER_ENDPOINT}
-        port = 443
+        port = 80
```
Comment (RomanWlm) [inline src/it/resources/application.conf:142]:
```suggestion
        port = 443
```

File: wd-local.env [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+export WALLET_ETC_EXPLORER_VERSION=v3
+export WALLET_ETC_EXPLORER_PROXYUSE=false
+export FEES_ETC_PATH=/blockchain/v3/eth/fees
+
+
+
+
+
+
+
```
Comment (RomanWlm) [inline wd-local.env:44]:
```suggestion
```

File: src/main/resources/application.conf.sample [hunk 4]
```diff
@@ -121,42 +85,46 @@ explorer = {
...
         disable_sync_token = ${?DEFAULT_DISABLE_SYNC_TOKEN}
       }
       {
         currency = bitcoin
-        host = "https://api.ledgerwallet.com"
+        host = "https://explorers.api.vault.ledger.com"
         host = ${?WALLET_BTC_EXPLORER_ENDPOINT}
         port = 443
         port = ${?WALLET_BTC_EXPLORER_PORT}
         explorer_version = "v2"
```
Comment (RomanWlm) [inline src/main/resources/application.conf.sample:101]:
```suggestion
        explorer_version = "v3"
```

**Review by dud225 — COMMENTED** [review]
I fully support tthis endeavour :+1: 
I've 2 comments:
- We may need to configure the WD to point to a different explorer endpoint for all coins (eg to test a new infrastructure). There are a lot of coins, we may miss some of them. We would need to be able to overwrite the default settings.
- There may always be a gap between the coins defined within the WD and coins coming in on the Vault. Many coins have similarities with existings ones eg BTC, so it would be great to be able to dynamically add new coin settings in some way.

**Pull request merged**